src/vi/truyenqq/src/eu/kanade/tachiyomi/extension/vi/truyenqq/TruyenQQ.kt

```diff
@@ -26,7 +26,7 @@ class TruyenQQ : ParsedHttpSource() {
 
     override val lang: String = "vi"
 
-    override val baseUrl: String = "https://truyenqqmoi.com"
+    override val baseUrl: String = "https://truyenqqq.vn/"
```

Comment:
Remove the trailing slash.

```suggestion
    override val baseUrl: String = "https://truyenqqq.vn"
```